src/Compatibility/Microsoft.DotNet.PackageValidation/ValidatePackage.cs

```diff
@@ -30,6 +33,26 @@ public class ValidatePackage : TaskBase
 
         public string CompatibilitySuppressionFilePath { get; set; }
 
+        public string RoslynAssemblyPath { get; set; }
+
+        public override bool Execute()
+        {
+            if (string.IsNullOrEmpty(RoslynAssemblyPath))
```

Comment:
Should we add the `Required` attribute to the parameter so that if it is empty rather than throwing an exception we get an MSBuild error saying that parameter is required?